Data/Base.rte/Shaders/ScreenBlit.frag

```diff
@@ -20,5 +20,5 @@ void main() {
 	vec4 guiColor = texture2DAA(rteGUITexture, textureUV);
 	float guiSolid = step(0.00000001, guiColor.r + guiColor.g + guiColor.b);
 	float blendRatio = max(guiColor.a, guiSolid);
-	FragColor = (texture2DAA(rteTexture, textureUV) * (1 - blendRatio)) + guiColor * blendRatio;
+	FragColor = (texture2DAA(rteTexture, textureUV) * (1.0 - blendRatio)) + guiColor * blendRatio;
```

Comment:
Why not 1.0F?

Reply:
glsl implicitly uses float for everything, specfying f is redundant